packages/tests/src/lib/service_node.ts

```diff
@@ -255,7 +283,7 @@ export class ServiceNode {
     ]);
   }
 
-  async messages(
+  async messagesRpc(
```

Comment:
nit: do we need to update the new function name within any tests?

Reply:
do we even need to keep it?

Reply:
removed